source/publish_project/deploy/index.html.md.erb

```diff
@@ -1,41 +1,35 @@
 ---
-title: "Deploy your documentation site"
-weight: 52
+title: Deploy and host your documentation site
+weight: 71
 last_reviewed_on: 2019-10-03
 review_in: 1 day
 ---
 
-# Deploy your documentation site
+# Deploy and host your documentation site
 
 The Tech Docs Template is built on Middleman, which is a static
 site generator. You can therefore deploy your site anywhere that supports
 static sites.
 
 ## Use the GOV.UK PaaS
 
-We recommend that government services use the [GOV.UK
-PaaS](https://www.cloud.service.gov.uk/) to deploy documentation sites built
-with the Tech Docs Template. This is also free of charge for
-government services.
+We recommend that government services use the [GOV.UK PaaS][paas] to deploy documentation sites built with the Tech Docs Template. This is also free of charge for government services.
 
 ## GitHub Pages
 
-With some modification, you can also deploy your site with [GitHub
-Pages](https://pages.github.com/), but we do not support this. To do this, you
-could for example use the [`middleman-gh-pages`]
-tool](https://github.com/edgecase/middleman-gh-pages), which we do not
-support. We also cannot guarantee that all features of the template will work if
-you deploy your site with GitHub Pages.
+With some modification, you can also deploy your site with [GitHub Pages][gh-pages], but we do not support this. To do this, you could for example use the [`middleman-gh-pages` tool][middleman-gh-pages], which we do not support.
+
+We also cannot guarantee that all features of the template will work if you deploy your site with GitHub Pages. The [Ministry of Justice cloud platform documentation][gh-pages-example] uses GitHub pages.
```

Comment:
It seems odd that the Ministry of Justice sentence follows straight on from the 'We cannot guarantee...' sentence. Should it be a separate paragraph?

The 'p' in 'GitHub pages' also needs capitalising.